scripts/write_gcr_to_parquet.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
+
+"""
+Write a GCR Catalog out to a Parquet file.
+
+pandas  # version >= 0.21
+generic-catalog-reader
+LSSTDESC/gcr-catalog
+
+and either
+pyarrow or fastparquet.
+"""
+
+
+import os
+import sys
+
+import pandas as pd
+
+import GCRCatalogs
+
+
+def convert_cat_to_parquet(reader='dc2_coadd_run1.1p',
```

Comment:
How about making the function arguments consistent with CLI arguments?

```suggestion
def convert_cat_to_parquet(reader,
```